config/default/manager_auth_proxy_patch.yaml

```diff
@@ -23,3 +23,4 @@ spec:
         args:
         - "--metrics-bind-addr=127.0.0.1:8080"
         - "--leader-elect"
```

Comment:
@mkumatag any specific reason for adding this manager here in this file, Can we remove it. The reason is it will be more meaningful to add - `"--powervs-provider-id-fmt=${POWERVS_PROVIDER_ID_FORMAT:=v1}"` to manager.yaml, But because of this, Its being overwritten and we are forced to add the flag here. cc: @Amulyam24

Reply:
I don't mind, may be we need to remove these args from this file then!

Reply:
yes, it can be removed.